index.js

```diff
@@ -54,6 +54,40 @@ function monkeypatch() {
     estraverse.VisitorKeys.TypeAlias = TypeAliasKeys;
     return results;
   };
+
+  // monkeypatch escope/referencer
+  var referencerLoc;
+  try {
+    var referencerLoc = Module._resolveFilename("./referencer", escopeMod);
+  } catch (err) {
+    throw new ReferenceError("couldn't resolve escope/referencer");
+  }
+  var referencer = require(referencerLoc);
+
+  // if there are decotators, then visit each
+  function visitDecorators(node) {
+    if (node.decorators) {
```

Comment:
Sure sounds good. Actually wanted to make a jscs rule for this https://github.com/jscs-dev/node-jscs/issues/1223 - not sure how to implement yet.